### PR TITLE
i18n(zh-cn): update vercel.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

update vercel.mdx

#### Related issues & labels (optional)

- Closes #
- Suggested label: 
https://github.com/withastro/docs/pull/8217

### CODE REVIEW

File: src/content/docs/zh-cn/guides/deploy/vercel.mdx [hunk 1]
```diff
@@ -92,7 +92,3 @@ npx astro add vercel
 你可以使用 `vercel.json` 来覆盖 Vercel 的默认行为并配置其他设置。例如，你可以在部署时为 HTTP 响应附加标头。
 
 <ReadMore>详细请参考 Vercel 文档的 [项目配置](https://vercel.com/docs/project-configuration)。</ReadMore>
-
-### 升级到 Astro 2.0
-
-Astro v2.0 不再支持 Node 14，因此请确保你的项目使用 **Node `18.14.1` 或更高版本**。你可以在项目设置的 “通用” 页面中的 “构建步骤和无服务器函数” 下 [自定义所使用的 Node.js 版本](https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings)。
```
Comment (hexWars) [inline src/content/docs/zh-cn/guides/deploy/vercel.mdx:98]: vercel的用户界面是英文，是否将`通用`修改成`General`更合适

**Review by CGQAQ — APPROVED** [review]

**Review by Nin3lee — APPROVED** [review]
LGTM!

**Review by hexWars — APPROVED** [review]
LGTM!

**Review by yuhangch — APPROVED** [review]
LGTM

**Review by emjio — APPROVED** [review]
LGTM

**Pull request merged**